### DISCUSSION TITLE
Backdrop-brightness doesn't work when background is set

### DISCUSSION THREAD

**Discussion by danileph** [tailwindlabs/tailwindcss#11863] [Help] [answered]
I'm using Next.js 13 with tailwind css. While creating a custom button component, I've encountered with a problem: backdrop-brightness doesn't work when background is set. I want the button have a background color and when a user hover the cursor on it, the button is supposed to become lighter. However, when the bg-primary isn't set, backdrop-brightness-125 works as expected.

```typescript     
const Button = () => {
  return (
      <button
        className={twMerge(
          "border-darkest border-2 rounded-xl px-4 py-2 uppercase select-none text-sm font-semibold tracking-widest bg-primary backdrop-brightness-125"
        )}
      >
        button
      </button>
  )
}
```

**Comment by wongjn** [comment, marked as answer]
The `backdrop-*` set of classes applies the `backdrop-filter` CSS property, and as per [the MDN documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter):

> The `backdrop-filter` [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS) property lets you apply graphical effects such as blurring or color shifting to the area _behind_ an element. Because it applies to everything behind the element, to see the effect you must make the element or its background at least partially transparent.

So to have `brightness()` work on the element itself, consider using a [`brightness-*`](https://tailwindcss.com/docs/brightness) class. Furthermore, to have it apply on hover only, use the `hover:` variant prefix:

```
hover:brightness-125
```

See an example on [Tailwind Play](https://play.tailwindcss.com/HY6BIvPbpx).
